packages/grid/test/data-provider.test.js

```diff
@@ -1,6 +1,8 @@
 import { expect } from '@vaadin/chai-plugins';
 import { aTimeout, fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
+import '../all-imports.js';
+import './data-provider.styles.js';
```

Comment:
Let's move these imports to get rid of the warning with Lit version:


```suggestion
import './data-provider.styles.js';
import '../all-imports.js';
```

```
 🚧 Browser logs:
      The custom element definition for "vaadin-grid" was finalized before a style module was registered. Ideally, import component specific style modules before importing the corresponding custom element. This warning can be suppressed by setting "window.Vaadin.suppressPostFinalizeStylesWarning = true".
```

Reply:
Done.